audioscrape/audioconvert.py

```diff
@@ -0,0 +1,17 @@
+# coding=utf-8
+'''Convert audio clips to user defined format via ffmpeg'''
+import subprocess
+
+
+def ffmpeg_convert(file, audio_title, fileformat):
+    command = ['ffmpeg', '-hide_banner',  # quiet ffmpeg banner
+               '-loglevel', 'panic',   # quiet ffmpeg stdout
+               '-i', "./{}".format(file),  # input file to convert
+               '-f', '{}'.format(fileformat),  # output fileformat type
+               '-ac', '1',  # mono channel
+               '-ar', '16000',  # sampling rate 16000Hz
```

Comment:
Should 16 kHz mono be the default or is 44100 kHz / 16-bit stereo more common? What are the intended use cases of the audio conversion?

Reply:
44100 Hz stereo is better quality, but the specific application I'm using your scraper for is for ASR model training which are generally built for 16000 Hz mono channel wav files. I should probably re-tool it to allow user defined settings, but admit I just added a quick and dirty hard coded solution for my needs